lib/reek/smells/nested_iterators.rb

```diff
@@ -72,9 +71,10 @@ def find_iters_for_iter_node(exp, depth)
         result
       end
 
+      # :reek:FeatureEnvy
```

Comment:
FeatureEnvy, but as far as I can see this is a good and clean solution, so I'd rather leave it like that. (we also might consider configuring FeatureEnvy to be a little bit more lenient, but that's another story).